frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx

```diff
@@ -112,7 +113,7 @@ import ExposureAnalysisTable from './AnalysisTable/ExposureAnalysisTable';
 import ExposureAnalysisActions from './ExposureAnalysisActions';
 import { getExposureAnalysisTableData } from '../../utils';
 
-const tabIndex = 2;
+const tabIndex = Panel.Analysis;
```

Comment:
To make it clearer it might be better to change from "index" to "panelId" or something like that? `index` is used in a few places